.docs/summary-generator.cjs

```diff
@@ -1,14 +1,13 @@
 const path = require("path");
 const fs = require("fs");
-const docsDir = "docs";
-const directoryPath = path.join(__dirname, "..", docsDir);
 
-async function prepareDocAnchor(type, file) {
+async function prepareDocAnchor(docsDir, type, file) {
+  const directoryPath = path.join(docsDir);
   const filePath = path.join(directoryPath, type, file);
   const fileContent = await fs.readFileSync(filePath, "utf-8");
```

Comment:
'await' is unnecesary here, but it was here before, but it can be improved by the way..